crates/evm/core/src/constants.rs

```diff
@@ -37,6 +37,11 @@ pub const DEFAULT_CREATE2_DEPLOYER_DEPLOYER: Address =
     address!("3fAB184622Dc19b6109349B94811493BF2a45362");
 /// The default CREATE2 deployer.
 pub const DEFAULT_CREATE2_DEPLOYER: Address = address!("4e59b44847b379578588920ca78fbf26c0b4956c");
+
+/// The default CREATE2 deployer for zksync (0x0000000000000000000000000000000000010000)
```

Comment:
let's also put a link to the discussion, and perhaps have ML document it in their docs so we can point to it instead

Reply:
also, we should move this to our crates and depend on that, otherwise we will get even more conflicts... it's gonna make it more difficult to check below for the output parsing - in that case then it's fine to leave as it, I'd rather keep this separate

Reply:
Yeah this should be in one of our zksync core crates